src/components/help/help.component.js

```diff
@@ -84,6 +84,9 @@ const Help = ({
         tooltipBgColor={tooltipBgColor}
         tooltipFontColor={tooltipFontColor}
         tooltipFlipOverrides={tooltipFlipOverrides}
+        ariaLabel={ariaLabel || type}
+        role="tooltip"
+        blockFocusBehaviour
```

Comment:
`blockFocusBehaviour` is making this focusable. I think it should be opt-out, so focusable by default, and opt-out on form validations, what do you think?

Reply:
Yeah I'll change it to be `focusable` or something similar